### ISSUE TITLE
ModelTrait not compactable with BaseModel CI4.5

### ISSUE THREAD

**Issue by seunex17** [tattersoftware/codeigniter4-relations#51]
This package does not seem to work for Codeigniter V4.5

**Comment by vonboth** [comment]
@MGatner 
Can you pls. update this package to work with CI4.5? Thank you!

**Comment by seunex17** [comment]
@vonboth i already fix this and send a pull request maybe he can check. 
https://github.com/tattersoftware/codeigniter4-relations/pull/52

**Comment by damian-romanowski** [comment]
Can we get a semver release now that #52 has been merged in?

Composer is still on v2.1.1.

**Comment by vonboth** [comment]
@MGatner
Can you please create a new release?

**Comment by vonboth** [comment]
@MGatner
Can I support you somehow with the upgrade? What is missing?

Because you said you don't use the project any longer would you like to give it to someone else?

**Comment by MGatner** [comment]
I'm sorry for the delay. I went to make the release and got stuck trying to fix pipeline errors. I think the issues aren't relevant, so I'll put the question to you all who are actively using this: ignore the pipeline errors and proceed with release, or someone dig into the issues blocking the pipeline?

**Comment by damian-romanowski** [comment]
The issues don't seem relevant.

I've been patching my installs with `sed -i 's/findAll(int $limit = 0, int $offset = 0)/findAll(int $limit = null, int $offset = 0)/g' vendor/tatter/relations/src/Traits/ModelTrait.php` after composer install and all works fine on ^4.5.

**Comment by vonboth** [comment]
I agree with you both. It doesn't seem to be relevant. So I 'd appreciate to see a new release soon.

**Comment by win123139** [comment]
no one fix it ?

**Comment by warcooft** [comment]
it looks like @MGatner are quite busy, 
as an alternative you guys can use the nested model package which already includes what this package provides.
see https://github.com/michalsn/codeigniter-nested-model
